crane_x7_examples_py/crane_x7_examples_py/joint_values.py

```diff
@@ -48,7 +48,9 @@ def main(args=None):
     )
 
     # 動作速度の調整
-    arm_plan_request_params.max_acceleration_scaling_factor = 0.5  # Set 0.0 ~ 1.0
+    arm_plan_request_params.max_acceleration_scaling_factor = (
+        0.5  # Set 0.0 ~ 1.0
+    )
     arm_plan_request_params.max_velocity_scaling_factor = 0.5  # Set 0.0 ~ 1.0
```

Comment:
こちらS17同様このレイアウトでいいかと思います。
```suggestion
    # Set 0.0 ~ 1.0
    arm_plan_request_params.max_acceleration_scaling_factor = 0.5
    arm_plan_request_params.max_velocity_scaling_factor = 0.5
```